packages/e2e/test/wallet/SingleAddressWallet/staking-rewards.test.ts

```diff
@@ -1,8 +1,8 @@
 import * as envalid from 'envalid';
```

Comment:
Please also update the `describe` block. It currently says 'Local Network'

Reply:
I think "delegation rewards" is a better term